packages/astro/src/vite-plugin-astro/hmr.ts

```diff
@@ -59,11 +60,12 @@ export interface HandleHotUpdateOptions {
 	config: AstroConfig;
 	logging: LogOptions;
 	compile: () => ReturnType<typeof cachedCompilation>;
+	viteServer: ViteDevServer;
 }
 
 export async function handleHotUpdate(
 	ctx: HmrContext,
-	{ config, logging, compile }: HandleHotUpdateOptions
+	{ config, logging, compile, viteServer }: HandleHotUpdateOptions
```

Comment:
Do we need to pass the `viteServer` in on its own here? `ctx: HmrContext` should include `server: ViteDevServer`

Reply:
Not sure I understand the suggest. Do you mean to pass the `ctx` instead?

Reply:
Oh, it already is passed, right.

Reply:
Fixed